protected/humhub/modules/user/models/User.php

```diff
@@ -192,6 +193,17 @@ public function isAttributeRequired($attribute)
         return parent::isAttributeRequired($attribute);
     }
 
+    /**
+     * Validate attribute username
+     * @param string $attribute
+     */
+    public function validateForbiddenUsername($attribute, $params)
+    {
+        if (in_array(strtolower($this->$attribute), Yii::$app->controller->module->forbiddenUsernames)){
+            $this->addError($attribute, Yii::t('UserModule.base', 'You cannot use this username.'));
```

Comment:
@vsemradu Please use message category `UserModule.account` here. (Same as in AccountChangeUsername)

Reply:
done